app/src/main/java/com/android/unio/model/firestore/transform/Serialization.kt

```diff
@@ -51,11 +51,12 @@ private fun mapUsersToRoles(members: List<Member>): Map<String, String> {
  * @param roles Roles to map.
  * @return Map of role UIDs to role data.
  */
-private fun mapRolesToPermission(roles: List<Role>): Map<String, Map<String, Any>> {
+fun mapRolesToPermission(roles: List<Role>): Map<String, Map<String, Any>> {
   return roles.associate { role ->
     role.uid to
         mapOf(
             Role::displayName.name to role.displayName,
+            Role::color.name to role.color,
             Role::permissions.name to
                 role.permissions.getGrantedPermissions().map { it.stringName })
   }
```

Comment:
This should be moved in a Utils file, but you already told me that you were planning on doing it